apps/doji-frontend/src/pages/profile/[username]/index.tsx

```diff
@@ -94,9 +109,16 @@ const ProfilePage: React.FC<ProfilePageProps> = ({ user }) => {
       />
       {currentUser.username !== displayUser.username && (
         <Stack spacing={3} direction={'row'}>
-          <Button fullWidth onClick={addFriend}>
-            <FiUserPlus style={{ marginRight: 8 }} /> Add Friend
-          </Button>
+          {relation == 'friend' ? (
+            <Button fullWidth disabled variant="outlined">
+              <FiCheck style={{ marginRight: 8 }} /> Friend
+            </Button>
+          ) : (
+            <Button fullWidth onClick={() => addFriend(username)}>
+              <FiUserPlus style={{ marginRight: 8 }} /> Add Friend
+            </Button>
```

Comment:
Button มี props สำหรับใส่ icon อยู่ ดูตามนี้เลย

https://mui.com/material-ui/react-button/#buttons-with-icons-and-label